prusa/link/web/lib/files.py

```diff
@@ -80,15 +80,17 @@ def sdcard_refs():
     }
 
 
-def gcode_analysis(meta):
+def gcode_analysis(meta, sd: bool = False):
     """Make gcodeAnalysis structure from metadata."""
+    if not sd:
+        meta = meta.data
```

Comment:
why? that must be fixed from place where gcode_analysis is called

Reply:
Fixed